README.md

```diff
@@ -0,0 +1,87 @@
+
+## Callbacks to the Rescue
+
+Now that we've accomplished our first objective -- learning how to explain what an asychronous process is -- let's examine the most basic and common method for handling asynchrony using a pattern known as a "callback." Since you've probably already used jQuery quite a bit, it is very likely that you've already written a few callback functions, perhaps without knowing it. If so, maybe something of this will feel familiar!
+
+Before we jump into an example of a callback, let's imagine a more real-world async example. Instead of making a sandwich, let's think about fetching search results from a server and displaying them to a user. All we need to  do is the following:
+
+```
+1. Send the search terms to the server that will fetch our results.
+3. Display the results.
+```
+
+This is simple indeed, but the fetching of results from the server -- just as the gathering of ingredients in our peanut butter sandwich example -- will take an indeterminate amount of time. So how do we handle this scenario?
+
+The most basic way to handle this problem is to use a "callback", and a callback is just a function that is called ("back") once an asynchronous process has completed. You can think of it this way: if you call someone and ask them do something for you that might take some time, you might also ask them to give you a "call back" when they are done so that 1) you know they are done, and 2) they can give you any information that you might need so that you can move forward with your part of the task. That's really all a callback is, and when we write them in our code all we are doing is explicitly organizing this process of coordination as part of the logic of the flow of our application.
+
+Enough theory. Here's how we would use callbacks inside a jQuery AJAX request to perform a search on Google's Books API for works related to peanut butter sandwiches:
+
+```
+var successCallback = function(data) {
+  // Display results on the page.
+};
+
+var errorCallback = function() {
+  alert('Oh no! Something went wrong while fetching results from the server.');
+}
+
+function fetchResults(searchTerm) {
+  $.ajax({
+    method: 'GET',
+    url: 'http://www.googleapis.com/books/v1/volumes?q=' + searchTerm,
+    success: successCallback,
+    error: errorCallback
+  });
+}
+
+fetchResults('peanut butter sandwich');
+```
+
+So let's look at this code. The first two var declarations create two callback functions: `successCallback` and `errorCallback`. Then we define a function `fetchResults` that contains a jQuery AJAX request. The jquery ajax method takes an object with some options set: the first two setup the request itself by specifying the url and the method. Then there are two additional parameters `success` and `error` and here is where we link in our callback functions. Now, once this request returns back either sucessfully or with an error, the appropriate callback function will be called.
```

Comment:
"jquery ajax" -> "`jQuery.ajax()`" (maybe?)
"the first two setup" -> "the first two set up"
"two additional parameters `success` and `error` and here..." -> "two additional parameters, `success` and `error`, and here..."